projects/hipblaslt/CMakeLists.txt

```
@@ -116,6 +116,7 @@ if(HIPBLASLT_ENABLE_HOST)
     endif()
     if(HIPBLASLT_ENABLE_MARKER)
         find_library(rocTracer roctx64 REQUIRED)
+        find_path(ROCTRACER_INCLUDE_DIR "roctracer/roctx.h" REQUIRED)
```

Can you expand on what the installation looks like in Spack? re:
> installed in seperate paths so the roctracer include directory needs to be explicitly added.

There is potential for rocTracer to find these headers inconsistently if multiple installations are available. Probably best to preferentially search from the same installation path. E.g.,
```suggestion
        get_filename_component(ROCTRACER_LIB_DIR ${rocTracer} DIRECTORY)
        get_filename_component(ROCTRACER_ROOT_DIR ${ROCTRACER_LIB_DIR} DIRECTORY)
    
        find_path(ROCTRACER_INCLUDE_DIR "roctracer/roctx.h" 
            HINTS ${ROCTRACER_ROOT_DIR}
            PATH_SUFFIXES include
            REQUIRED)
```
Even if Spack ensures a single installation, this will also affect monolithic installs.

---

Thanks, your suggestion looks good. The only thing we need for Spack is the include directories added and visible.